web/src/lib/components/forms/admin-registration-form.svelte

```diff
@@ -16,7 +17,7 @@
 
   $: {
     if (password !== confirmPassword && confirmPassword.length > 0) {
-      errorMessage = 'Password does not match';
+      errorMessage = $t('component.form.error_password_mismatch');
```

Comment:
Just curious - did you run a codemod or regexp to find all the strings?

Reply:
Up until now: Open File -> look around -> Copy/Paste.

As it was just some testing to get a feel if/how that works, that was alright. For further work on this I thought about trying to use a regexp and small script to at least get most of the strings automatically into a file.